pkg/operator/ceph/cluster/watcher.go

```diff
@@ -243,6 +245,10 @@ func (c *clientCluster) fenceNode(ctx context.Context, node *corev1.Node, cluste
 				if err == nil {
 					break
 				}
+				// continue to fence next rbd volume if active client not found
+				if stderrors.Is(err, errActiveClientNotFound) {
+					continue
+				}
```

Comment:
PR title says cephfs. This seems to be operating on RBD PVs. Is this intended?

Reply:
The PR was originally just for cephfs, but also expanded for rbd. I've updated the title.